content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,106 @@
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
+* [Bookinfo with a Virtual Machine](/docs/examples/virtual-machines/) to learn more about connecting virtual machine workloads to Kubernetes workloads.
+
+## Request Classification (Beta)
+
+Istio continues to make mesh telemetry collection more configurable. In this release,
+[Request Classification](/docs/tasks/observability/metrics/classify-metrics/) has been promoted to Beta. This feature
+enables users to more precisely understand and monitor the traffic in their service mesh.
+
+## Kubernetes Service API support (Alpha)
+
+Support for [Kubernetes `service-apis`](https://kubernetes-sigs.github.io/service-apis/) has been an active area of
+development since Istio 1.8. These are marketed as the next generation of the Ingress API, and draw inspiration from
+Istio's `Gateway` and `VirtualService`. The Istio community is eager to evolve with Kubernetes and these CRDs in upcoming
+releases. To try them out, check out the [getting started documentation](/docs/tasks/traffic-management/ingress/service-apis/).
+These APIs are still evolving, so feedback is welcome!
+
+## Better external authorization (Experimental)
+
+Authorization policy now supports an experimental feature of
+[`CUSTOM` action](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action) that allows users to
+integrate with external auth system (e.g. OPA, OAuth2, etc.) more easily.
+
+We will soon be publishing a blog on better external authorization, but for now you can look at our documentation,
+[external authorization with custom action task page](/docs/tasks/security/authorization/authz-custom) to use this
+feature. If you are using envoy filter today to integrate with an external auth system, we would recommend you try this
+feature out and give us feedback!
+
+## Mirroring of images on `gcr.io`
+
+To prevent our users from getting affected by Docker Hub's [rate-limiting policy](/blog/2020/docker-rate-limit/),
+we are now publishing all our images on the `gcr.io/istio-release`registry. You can optionally set the hub in your
+installation step to `gcr.io/istio-release` to get around issues related to failed image downloads from Docker hub. Note
+that Docker hub is still the default hub for Istio installation.
+
+## istioctl updates
+
+We have continued to make significant improvements in the `istioctl` tool to improve the troubleshooting and debugging
+capabilities for our users. Key features include:
+* A new `verify-install` command that notifies users of any installation configuration errors.
+* The `analyze` subcommand can now check if deprecated or alpha-level annotations are used.
```

Comment:
@nmittler 

have we updated the annotation doc page to capture status? If so could link from here

Reply:
Yes, the current annotations page page is [here](https://preliminary.istio.io/latest/docs/reference/config/annotations/)

Reply:
Added a link